package.json

```diff
@@ -2,22 +2,14 @@
   "name": "whodat",
   "private": true,
   "dependencies": {
-    "@hotwired/stimulus": "^3.0.1",
-    "@hotwired/stimulus-webpack-helpers": "^1.0.1",
-    "@hotwired/turbo": "^7.0.1",
+    "@hotwired/stimulus": "^3.2.1",
+    "@hotwired/turbo": "^7.3.1",
```

Comment:
This is weird - Turbo doesn't have a 7.3.1 version, so yarn won't install without manual intervention. The v7 series release is 7.3.0.

Reply:
I've tidied this up.